src/core_tests.rs

```diff
@@ -38,6 +38,9 @@ use std::sync::mpsc;
 use std::thread;
 use types::MessageId;
 use xor_name::XorName;
+use maidsafe_utilities;
+// Add the following line to tests to enable logging:
```

Comment:
But they are _commonly_ used to debug tests, and this saves looking up the log-init line.